app/src/main/res/layout/activity_login.xml

```diff
@@ -6,4 +6,60 @@
     android:layout_height="match_parent"
     tools:context=".ui.login.LoginActivity">
 
+    <TextView
+        android:id="@+id/textView"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/dp_128"
+        android:text="@string/title_login"
+        android:textSize="@dimen/sp_24"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.496"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <EditText
+        android:id="@+id/editAccount"
+        android:layout_width="@dimen/sp_270"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/dp_68"
+        android:layout_marginTop="@dimen/dp_72"
+        android:ems="10"
+        android:hint="@string/hint_account"
+        android:inputType="textPersonName"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/textView" />
+
+    <EditText
+        android:id="@+id/editPassword"
+        android:layout_width="@dimen/sp_270"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/dp_68"
+        android:layout_marginTop="@dimen/dp_20"
+        android:ems="10"
+        android:hint="@string/hint_password"
+        android:inputType="textPassword"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/editAccount" />
+
+    <Button
+        android:id="@+id/buttonSignIn"
+        android:layout_width="@dimen/sp_270"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/dp_68"
+        android:layout_marginTop="@dimen/dp_28"
+        android:text="@string/button_sign_in"
+        android:background="#00BB00"
```

Comment:
để ra file colors.xml.
Đặt tên cho màu cần rõ nghĩa theo chức năng hoặc có thể tham khảo tên bằng tool này
http://chir.ag/projects/name-that-color